mitmproxy/utils/typecheck.py

```diff
@@ -26,7 +26,7 @@ def check_type(attr_name: str, value: typing.Any, typeinfo: type) -> None:
     if typename.startswith("typing.Union"):
         try:
             types = typeinfo.__args__
-        except AttributeError:
+        except AttributeError:  # pragma: no cover
```

Comment:
This branch is not executed on Python 3.6 - because the import works (and does not fail as on Python 3.5).